py/src/nupic/tests/algorithms/anomaly_likelihood_jeff_test.py

```diff
@@ -35,7 +35,7 @@
 import unittest
 
 from nupic.algorithms import anomaly_likelihood as an
-from nupic.support.unittesthelpers.testcasebase import TestCaseBase
+from unittest import TestCase as TestCaseBase
```

Comment:
I like this change! Makes the diff much smaller, and I agree we should limit the scope of things we care about. 
Thanks :+1: